oci/main.py

```diff
@@ -16,12 +17,17 @@ def run():
         '--debug',
         help="Show noisy debug logs",
         action="store_true")
+
+    parser.add_argument(
+        '--daemon',
+        help="Run as daemon and redirect output to /var/tmp/oci_{command}_{change id}.log",
+        action="store_true")
```

Comment:
This option is reasonable, but using /var/tmp/oci_... will create lot of junk logs that someone will have to cleanup. This is a usability problem.

Also using change id, later builds will override older builds logs, hiding errors you may want to inspect.

Reply:
can add timestamp to file name as well